src/components/Map.js

```diff
@@ -76,12 +77,14 @@ const Map = ({
 
   return (
     <div className="map-container" hidden={isMobile && mobileView !== 'map'}>
-      <div className="logo">
-        <img
-          src="/images/bikesy-logo.png"
-          srcSet="/images/bikesy-logo@2x.png 2x"
-          alt="logo"
-        />
+      <div className={appConfig.LOGO_CLASSNAME}>
+        <a href={appConfig.ABOUT_LINK_URL}>
```

Comment:
The Tahoe map had a hyperlink around the logo but the bikesy map didn't. I added a hyperlink to the about page. Do we want this?